src/docker-utils.coffee

```diff
@@ -127,9 +129,10 @@ do ->
 					supervisorTags = _.flatten(supervisorTags)
 					appTags = _.flatten(appTags)
 					locallyCreatedTags = _.flatten(locallyCreatedTags)
-					return { images, supervisorTags, appTags, locallyCreatedTags }
+					resinhupTagRegex = new RegExp("resinhup/resinhup-#{deviceType}$")
```

Comment:
That would require resinhup to have access to the supervisor API key, which it currently doesn't. It would also get us into an entanglement between the supervisor and resinhup which I'd like to avoid (as resinhup aggressively changes the underlying OS and will know best when to stop the supervisor, etc).
This hardcoding gives an easy win by allowing a shorter downtime, because the updater doesn't need to stop the supervisor while pulling the resinhup image.